sympy/stats/frv_types.py

```diff
@@ -305,6 +308,56 @@ def Binomial(name, n, p, succ=1, fail=0):
 
     return rv(name, BinomialDistribution, n, p, succ, fail)
 
+#-------------------------------------------------------------------------------
+# Beta-binomial distribution ----------------------------------------------------------
+
+class BetaBinomialDistribution(SingleFiniteDistribution):
+    _argnames = ('n', 'alpha', 'beta')
+
+    @staticmethod
+    def check(n, alpha, beta):
+        _value_check((n.is_integer, n.is_nonnegative),
+        "'n' must be nonnegative integer. n = %s." % str(n))
+        _value_check((alpha > 0),
+        "'alpha' must be: alpha > 0 . alpha = %s" % str(alpha))
+        _value_check((beta > 0),
+        "'beta' must be: beta > 0 . beta = %s" % str(beta))
+
+    @property
+    @cacheit
+    def dict(self):
+        n, a, b = self.n, self.alpha, self.beta
+        n = as_int(n)
+        return dict((k,
+                binomial(n, k) * beta_fn(k + a, n - k + b) / beta_fn(a, b)) for k in range(0, n + 1))
```

Comment:
```diff
-        return dict((k,
-                binomial(n, k) * beta_fn(k + a, n - k + b) / beta_fn(a, b)) for k in range(0, n + 1))
+        return dict((k, binomial(n, k) * beta_fn(k + a, n - k + b) / beta_fn(a, b))
+                     for k in range(n + 1))
```

Reply:
Split the line if longer than 79 characters.

Reply:
80 chars should still be fine, right?

Reply:
It could be. I am not sure how strictly we follow [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length):
> Limit all lines to a maximum of 79 characters.